source/kernel/src/lib.rs

```diff
@@ -366,12 +261,12 @@ impl Kernel {
         f(&mut guard)
     }
 
-    pub fn spawn_allocated<F>(&'static self, task: HeapBox<Task<F>>) -> JoinHandle<F::Output>
-    where
-        F: Future + 'static,
-    {
-        self.inner.scheduler.spawn_allocated::<F, HBStorage>(task)
-    }
+    // pub fn spawn_allocated<F>(&'static self, task: Box<Task<F>>) -> JoinHandle<F::Output>
```

Comment:
@hawkw do you have thoughts here about using the builtin `LocalScheduler` vs (re-)making a `LocalStaticScheduler` that could be allocated with the failable interface?

Reply:
I think this spawns tasks, not the scheduler. So if we spawned a new task 6 hours after startup, this *could* panic there.